packages/@ember/object/type-tests/computed/expand-properties.test.ts

```diff
@@ -0,0 +1,13 @@
+import { expandProperties } from '@ember/object/computed';
```

Comment:
I learned about this last week working on the DT update. It's... such a bananas thing. It makes sense that we have it, but I won't be sad to see it gone someday.